CIP-0011 | Adjust preamble and structure w.r.t CIP-0001 (#654)
cardano-foundation/CIPs: CIP-0011/README.md

stake1uy8ykk8dzmeqxm05znz65nhr80m0k3gxnjvdngf8azh6sjc6hyh36
```

## Rationale: how does this CIP achieve its goals?

### Meaning of *account*

The term "account" is unfortunately an overloaded term so we clarify all its uses here:

#### 1) "Account" as a BIP44 derivation level

BIP44 uses the term "account" as one derivation level to mean the following

> This level splits the key space into independent user identities, so the wallet never mixes the coins across different accounts.
To differentiate this from other usage, we sometimes refer to it as an `account'` (the bip32 notation) or a BIP44 Account.

#### 2) "Account" as a transaction model

Blockchains like Ethereum does not use the UTXO model and instead uses the [*Account model*](https://github.com/ethereum/wiki/wiki/Design-Rationale#accounts-and-not-utxos) for transactions.

## Path to Active

### Acceptance Criteria

> - [x] All notable wallet providers follow this method of key derivation.

Reviewer: ```suggestion
- [x] All notable wallet and tooling providers follow this method of key derivation.
```

Reviewer: just make a little wider

### Implementation Plan

> - [x] This method of key derivation has been agreed as canonical and made available to all wallet providers beginning with the Shelley ledger era.

Reviewer: ```suggestion
- [x] This method of key derivation has been agreed as canonical and has been included in [CIP-1852 | HD (Hierarchy for Deterministic) Wallets for Cardano](https://github.com/cardano-foundation/CIPs/blob/master/CIP-1852/README.md).

- [x] This method of key derivation has been supported by all wallet and tool providers beginning with the Shelley ledger era.
```

I like included the add to CIP-1852 as it is an important step IMO

Author: yes, good addition & correction; have committed... but had to put inter-CIP link into some canonical form to make sure it works on the Dev Portal which I think will not parse the absolute links into links also on the Dev Portal.  Otherwise your changes are the same. 🙏

## Copyright

> This CIP is licensed under [CC-BY-4.0](https://creativecommons.org/licenses/by/4.0/legalcode)

Reviewer: ```suggestion
This CIP is licensed under [CC-BY-4.0](https://creativecommons.org/licenses/by/4.0/legalcode).
```
I will never miss a `.` !!!

Author: I didn't miss them either, but when editing I decided in these cases when this particular full stop was missing to grant the author "poetic license" 😝 ... so we will probably get a dozen more of these over the course of the "remediation" 😅

Reviewer: ok ok 🤣
I will leave it to "poetic license" hehe
